src/main/kotlin/racingcar/Car.kt

```diff
@@ -0,0 +1,24 @@
+package racingcar
+
+import kotlin.random.Random
+
+data class Car(var distance: Int = 0) {
+
+    fun moveRandom(number: Int = Random.nextInt(Companion.MAX_RANDOM_VALUE)) {
+        if (canMove(number)) {
+            distance += 1
+        }
+    }
+
+    private fun canMove(number: Int): Boolean {
+        if (number >= 4) {
+            return true
+        }
+
+        return false
+    }
```

Comment:
```suggestion
    private fun canMove(number: Int) =
        number >= 4
```
if문을 사용하지 않고 간단하게 표현할 수 있어요